Amazon.IonDotnet.Tests/Common/ReaderTestCommon.cs

```diff
@@ -250,6 +278,14 @@ public static void HasAnnotationFalse_SingleField(IIonReader reader)
             Assert.IsFalse(reader.HasAnnotation("Spam Musubi"));
         }
 
+        public static void HasAnnotationTrue_ZeroSymbol(IIonReader reader)
+        {
+            // an int with zero symbol annotation
+            // $0::18
+            reader.MoveNext();
+            Assert.IsTrue(reader.HasAnnotation(null));
```

Comment:
`reader.HasAnnotation(null)` should always be false (or perhaps raise an exception). `null` is not synonymous with symbol zero.

Reply:
Revert to original, which was raising exception